docs/trace/customizing-the-sdk/README.md

```diff
@@ -195,8 +202,10 @@ exporting purposes, the SDK provides the following built-in processors:
   : This is an exporting processor which passes telemetry to the configured
   exporter without any batching.
 
-Follow [this](../extending-the-sdk/README.md#processor) document
-to learn about how to write own processors.
+Follow [this](../extending-the-sdk/README.md#processor) document to learn about
+writing custom processors. Follow
+[this](../extending-the-sdk/README.md#exporter) document to learn about writing
```

Comment:
We should probably include a blurb for exporters as well under `Processors & Exporters` section.

Reply:
say something like Exporters are not something TracerProvider is aware of...it only calls the processors, and it is the processor who invoke exporter.

Reply:
I will do this as a follow-up!